src/main/java/net/minecraftforge/fluids/Fluid.java

```diff
@@ -126,12 +127,26 @@
      */
     protected Block block = null;
 
+    /**
+     * @deprecated please specify an overlay texture that will be rendered on the screen when a player is submerged in your fluid. 
+     */
+    @Deprecated
     public Fluid(String fluidName, ResourceLocation still, ResourceLocation flowing)
     {
         this.fluidName = fluidName.toLowerCase(Locale.ENGLISH);
         this.unlocalizedName = fluidName;
         this.still = still;
         this.flowing = flowing;
+        this.overlay = new ResourceLocation("textures/misc/underwater.png");
```

Comment:
Just use `this(fluidName, still, flowing, new ResourceLocation("textures/misc/underwater.png"))`.